proto/beacon/rpc/v1/services.proto

```diff
@@ -27,7 +27,8 @@ service AttesterService {
 }
 
 service ProposerService {
-    rpc ProposeBlock(ProposeRequest) returns (ProposeResponse);
+    rpc ProposeBlock(ethereum.beacon.p2p.v1.BeaconBlock) returns (ProposeResponse);
+    rpc ComputeBlockWithStateRoot(ProposeRequest) returns (ethereum.beacon.p2p.v1.BeaconBlock);
```

Comment:
That is what I thought initially too but the proposer needs to sign the block before broadcasting it and it uses the beacon-node to obtain the `CandidatePowReceiptRootHash32` so that is why I pass the whole block back. I could add in another rpc method, just to retrieve the pow chain receipt root. This would keep the rpc responses light